lib/Capabilities.php

```diff
@@ -229,6 +230,7 @@ public function getCapabilities(): array {
 				'conversations' => [
 					'can-create' => $user instanceof IUser && !$this->talkConfig->isNotAllowedToCreateConversations($user),
 					'force-passwords' => $this->talkConfig->isPasswordEnforced(),
+					'compact-list' => $this->talkConfig->getConversationsListStyle($user?->getUID()),
```

Comment:
I wondered if a bool is the best option for now, or if we use "string" config already:
```suggestion
					'list-style' => $this->talkConfig->getConversationsListStyle($user?->getUID()),
```

And basically return the strings:
- `two-lines`
- `compact`

for now and then we could have other list styles in the future more easily?

Reply:
If boolean is already passed as 'yes' or 'no', then I don't see a reason to not do it as you suggesting from the beginning